### ISSUE TITLE
Dead?

### ISSUE THREAD

**Issue by James-Parsons** [couchbaselabs/nativescript-couchbase#44]
It seems the project hasn't been updated in over a month. Has this been abandoned? If not, is there any sort of roadmap for release?

**Comment by nraboy** [comment]
So a few things:

1. Because it is under **couchbaselabs** it is a community driven project.
2. NativeScript plugins by nature are only JavaScript wrappers to native features, SDKs, and libraries.  Not much has changed in the Couchbase iOS and Android SDKs so no need to nurse the wrapper on a weekly basis.
3. No one has opened tickets reporting issues which means either no one is using the plugin or there are no problems.  NPM stats say it is heavily used.

There is no roadmap because it isn't an officially supported plugin.  This doesn't mean it won't be in the future.

To sum it up, the plugin works great and I encourage you to contribute to it if you find something missing.

Best,
